test/server/models/Update.test.js

```diff
@@ -159,6 +159,13 @@ describe('server/models/Update', () => {
         expect(usersToNotify.length).to.eq(0);
       });
 
+      it('returns an empty array when the audience in NO_ONE', async () => {
+        const emptyCollective = await fakeCollective();
+        const update = await fakeUpdate({ CollectiveId: emptyCollective.id, notificationAudience: 'NO_ONE' });
```

Comment:
`emptyCollective` has no member, so this test doesn't serve its purpose. Let's use a collective with some data:

```suggestion
        const update = await fakeUpdate({ CollectiveId: collective.id, notificationAudience: 'NO_ONE' });
```